assets/src/edit-story/karma/textStyle.cuj.karma.js

```diff
@@ -0,0 +1,316 @@
+      await fixture.events.click(input);
+    };
+
+    beforeEach(async () => {
+      await openFontPicker();
+    });
+
+    it('should render the font picker', async () => {
+      await fixture.snapshot('font picker open');
+    });
+
+    it('should apply the selected font', async () => {
+      await fixture.events.keyboard.type('Yrsa');
+      // Ensure the debounced callback has taken effect.
+      await wait(TIMEOUT);
+      const option = fixture.screen.getByText('Yrsa');
+      await fixture.events.click(option);
+      await wait(TIMEOUT);
+      await openFontPicker();
+      const selected = fixture.screen.getAllByRole('option', {
+        name: 'Selected Yrsa',
+      });
+      expect(selected.length).toBe(1);
+
+      const {
+        state: {
+          currentPage: { elements },
+        },
+      } = await fixture.renderHook(() => useStory());
+      expect(elements[1].font.family).toBe('Yrsa');
+    });
+
+    it('should display only the fonts from curated list by default', () => {
+      const options = document
+        .getElementById('editor-font-picker-list')
+        .querySelectorAll('li');
+      expect(options.length).toBe(DEFAULT_VISIBLE_FONTS);
+    });
+
+    describe('when searching fonts', () => {
+      it('should display the correct fonts when searching', async () => {
+        await fixture.events.keyboard.type('Ab');
+        // Ensure the debounced callback has taken effect.
+        await wait(TIMEOUT);
+        let options = document
+          .getElementById('editor-font-picker-list')
+          .querySelectorAll('li');
+        expect(options.length).toBe(2);
+        expect(options[0].textContent).toBe('Abel');
+        expect(options[1].textContent).toBe('Abhaya Libre');
+
+        await fixture.events.keyboard.type('el');
+        // Ensure the debounced callback has taken effect.
+        await wait(TIMEOUT);
+        options = document
+          .getElementById('editor-font-picker-list')
+          .querySelectorAll('li');
+        expect(options.length).toBe(1);
+        expect(options[0].textContent).toBe('Abel');
+      });
+
+      it('should not search with less than 2 characters', async () => {
+        await fixture.events.keyboard.type('A');
+        // Ensure the debounced callback has taken effect.
+        await wait(TIMEOUT);
+        let options = document
+          .getElementById('editor-font-picker-list')
+          .querySelectorAll('li');
+        expect(options.length).toBe(DEFAULT_VISIBLE_FONTS);
+      });
+
+      it('should restore default fonts list when emptying search', async () => {
+        await fixture.events.keyboard.type('Ab');
+        // Ensure the debounced callback has taken effect.
+        await wait(TIMEOUT);
+        const options = document
+          .getElementById('editor-font-picker-list')
+          .querySelectorAll('li');
+        expect(options.length).toBe(2);
+
+        await fixture.events.keyboard.press('Del');
```

Comment:
This is failing for some reason and doesn't seem to delete the typed in character.

It passes locally but not here. @merapi @barklund If you have any thoughts on why this might be failing, please let me know.

Reply:
Issue resolved, had to use `Backspace` instead (props @barklund)

Reply:
Fixed after "Try using Backspace instead of Del", right? (MacOS Del vs Linux Del thing?)